src/devices/Ina219/Ina219.cs

```diff
@@ -6,20 +6,22 @@
 using System.Collections.Generic;
 using System.Device;
 using System.Device.I2c;
+using System.Device.Model;
 using System.Diagnostics.CodeAnalysis;
 using UnitsNet;
 
 namespace Iot.Device.Adc
 {
     /// <summary>
-    /// Binding that exposes an INA219 Bidirectional Current/Power monitor.
+    /// INA219 Bidirectional Current/Power monitor.
     ///
     /// The INA219 is a high-side current shunt and power monitor with an I2C interface.
     /// The INA219 monitors both shunt drop and supply voltage, with programmable conversion
     /// times and filtering. A programmable calibration value, combined with an internal multiplier,
     /// enables direct readouts in amperes. An additional multiplying register calculates power in watts.
     /// <see href="http://www.ti.com/lit/ds/symlink/ina219.pdf"/>
     /// </summary>
+    [Interface("INA219 Bidirectional Current/Power monitor")]
```

Comment:
are there any special character not allowed or which needs to be escaped? in this case the `/` ?